micrometer-core/src/test/java/io/micrometer/core/instrument/binder/kafka/KafkaClientMetricsIntegrationTest.java

```diff
@@ -90,6 +90,8 @@ void shouldManageProducerAndConsumerMetrics() {
         out.println("Meters from producer after sending and consumer before poll:");
         printMeters(registry);
 
+        Thread.sleep(5 * 1000);
```

Comment:
We could make a check first and then schedule to avoid the initial sleep?

Reply:
Yes, I think either doing the check as part of the logic or triggering the check in the test so we don't have to wait is better.

Reply:
If frequency were to be 1 min, then I’d prefer to do a first check before scheduling to have an initial set of metrics available

Reply:
Done :)

Reply:
For information, there is an initial fetch with the `scheduleAtFixedRate(Runnable command,
                                                  long initialDelay,
                                                  long period,
                                                  TimeUnit unit)`, I set an `initialDelay` to `0`